test/libman.Test/LibmanUpdateTest.cs

```diff
@@ -138,20 +138,15 @@ public void TestUpdateCommand_AlreadyUpToDate()
   ""defaultProvider"": ""cdnjs"",
   ""defaultDestination"": ""wwwroot"",
   ""libraries"": [
-    {
-      ""library"": ""jquery@3.5.0"",
-      ""files"": [ ""jquery.min.js"", ""jquery.js"" ]
-    }
   ]
 }";
 
             string libmanjsonPath = Path.Combine(WorkingDir, "libman.json");
             File.WriteAllText(libmanjsonPath, contents);
 
-            var restoreCommand = new RestoreCommand(HostEnvironment);
-            restoreCommand.Configure(null);
-
-            restoreCommand.Execute();
+            var installCommand = new InstallCommand(HostEnvironment);
+            installCommand.Configure(null);
+            installCommand.Execute("jquery --files jquery.min.js --files jquery.js".Split(' '));
```

Comment:
>("jquery --files jquery.min.js --files jquery.js".Split(' ')); [](start = 34, length = 62)

Looks like we are not specifying a version anywhere in the command? Should we?

Reply:
Not in this case.  This test is verifying that when the library is already up to date with the latest, then there should be no changes.  Because it restored using a constant version before, it would break every time a new version shipped (no longer restoring the latest version).

The change here is to remove the constant version and always install the latest, then run update and verify there's no change.